src/auto-complete.component.ts

```diff
@@ -105,7 +107,13 @@ import { NguiAutoComplete } from './auto-complete';
             border-bottom: none;
         }
 
-        .ngui-auto-complete > ul li:hover {
+        .header-item {
```

Comment:
this should be `.ngui-auto-complete > ul li.header-item` I think.

Reply:
Apologies, that .header-item selector was placed there during my testing and meant to be removed.  I will fix.